Scuti/Scripts/ScutiSDK/ScutiButton.cs

```diff
@@ -40,19 +42,58 @@ public void Start()
 
     public void OnClick()
     {
-        ScutiSDK.Instance.LoadUI();
-        NotificationIcon.SetActive(false);
+        if (canConnect)
+        {
+            ScutiSDK.Instance.LoadUI();
+            NotificationIcon.SetActive(false);
+        }
     }
 
     private async void CheckNewOffers()
     {
-        var stats = await ScutiAPI.GetCategoryStatistics();
-        NewItems?.SetActive(false);
-        if (stats != null)
+        try
         {
-            if (stats.NewOffers.HasValue && stats.NewOffers.Value > 0)
+            var stats = await ScutiAPI.GetCategoryStatistics();
+            NewItems?.SetActive(false);
+            if (stats != null)
             {
-                NewItems?.SetActive(true);
+                if (stats.NewOffers.HasValue && stats.NewOffers.Value > 0)
+                {
+                    NewItems?.SetActive(true);
+                }
+            }
+            canConnect = true;
+        }
+        catch (Exception)
+        {
+            Debug.LogError("couldn't find offers");
+            canConnect = false;
+            LoadToast();
+        }
+    }
+
+    private void LoadToast()
+    {
+        if(toast == null)
+        {
+            toast = Instantiate(Resources.Load<GameObject>(ScutiConstants.UI_TOAST_NAME));
+        }
```

Comment:
What is toast? And we are just instantiating it in their scene? What deletes it?

Reply:
Toast is the term for the message in the screen, in this case is a prefab with it's own canvas
It disables itself, but I can make it so it deletes